vagrant/provisioning/roles/foia/templates/arkcase-portal-server-2020.16.yaml

```diff
@@ -35,3 +35,9 @@ portal:
 
 
 configServerUri: "{{ internal_host }}/config"
+
+
+loginGov: {{ loginGov_authenticated_mode | default(false) }}
+Login.gov:
```

Comment:
Isn't there more to the login.gov configuration?  I remember it being many more lines.

Reply:
it seems the remaining config is standard for all sites, in arkcase-portal.yaml